docs/section19/19-14.md

```diff
@@ -0,0 +1,71 @@
+                - Get the zone's total return fan kW, need to get the second element in the list of fan powers:`modeled_return_fan_power_p = modeled_fan_power_list_p[1]` 
+                - Get the zone's total relief fan kW, need to get the fourth element in the list of fan powers:`modeled_relief_fan_power_p = modeled_fan_power_list_p[3]` 
+                - Check if the total return fan power is greater than 0 in the proposed, if yes then set is_modeled_with_return_fan_in_proposed to true: `if modeled_return_fan_power_p > 0: is_modeled_with_return_fan_in_proposed = true`  
+                - Check if the total relief fan power is greater than 0 in the proposed, if yes then set is_modeled_with_relief_fan_in_proposed to true: `if modeled_relief_fan_power_p > 0: is_modeled_with_relief_fan_in_proposed = true`  
+                            
+
+            - Create fan system object: `fan_sys_b = hvac.fan_system`  
+            - Get list from dictionary of the hvac system's fan system's supply, return, exhaust and relief fan CFM: `fan_sys_CFM = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"]["Fan_CFM"])`  
+            - Get the hvac system total supply fan cfm, need to get the first element in the list of fan cfms (this assumes that all fans associated with the supply_fans object are in parallel (i.e., if multiple fans the cfm is additive.):`hvac_sys_total_supply_fan_cfm_b = fan_sys_CFM[0]` 
+            - Calculate 90% of the supply air quantity: `90_percent_supply_cfm = 90% * hvac_sys_total_supply_fan_cfm_b`   
+            - Calculate the supply CFM minus the minimum OA CFM for the fan system: `supply_minus_OA_cfm = hvac_sys_total_supply_fan_cfm_b - fan_sys_b.minimum_outdoor_airflow`  
+            - Check if is_modeled_with_return_fan_in_proposed == true, if so then set the modeled_cfm equal to the return cfm (second element in list of baseline hvac fan cfms): `if is_modeled_with_return_fan_in_proposed == true: modeled_cfm = fan_sys_CFM[1]`  
+            - Else, is_modeled_with_relief_fan_in_proposed == true, then set the modeled_cfm equal to the relief cfm (fourth element in list of baseline hvac fan cfms): `Else: modeled_cfm = fan_sys_CFM[3]`   
+
+            - **Rule Assertion:** 
+            - Case 1: If return or relief fans are specified in the proposed design (but not both) and the baseline building design is modeled with fans serving the same functions and sized for the baseline system supply fan air quantity less the minimum outdoor air, or 90% of the supply fan air quantity, whichever is larger then pass: `if (all(is_modeled_with_return_fan_in_proposed, is_modeled_with_relief_fan_in_proposed) == false) and (is_modeled_with_return_fan_in_proposed == true or is_modeled_with_relief_fan_in_proposed == true) and (modeled_cfm == max(supply_minus_OA_cfm,90_percent_supply_cfm)): outcome = "PASS"`  
```

Comment:
What does "all" means in this context? That it is true for ALL zones served by that system?

I don't follow the meaning of the first two conditions in the if clause. Can you please explain?

The last condition (is_modeled_with_return_fan_in_proposed == true or is_modeled_with_relief_fan_in_proposed == true) allows for schenarios when both are true, which I don't think the intent.

Reply:
@mkarpman 
all() only returns true if both conditions are true so ensures that the last condition you mention above is not the case. 

This is the code: (all(is_modeled_with_return_fan_in_proposed, is_modeled_with_relief_fan_in_proposed) == false) so if this equals false then both of these are not true. Does this work or am I missing something?